clutch/src/main.rs

```diff
@@ -74,7 +135,8 @@ impl ReplTrait<F> for ClutchState<F> {
                                     if let Expression::Str(p) = store.fetch(&proof_path).unwrap() {
                                         p.to_string()
                                     } else {
-                                        panic!("proof path must be a string");
+                                        println!("Proof path must be a string");
```

Comment:
`eprintln!` sounds better at least, here and in the below.

Reply:
done